docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,89 @@ We can plot the calibration by calling::
     calibration.plot()
 
 .. image:: force_calibration_fit.png
+
+Hydrodynamics
+-------------
+
+While the idealized Lorentzian model discussed in the previous section provides a reasonable first description of the physical power spectrum, better models for this spectrum exist.
+
+The idealized model is based on the assumption that the drag force is only proportional to the bead's velocity.
+This assumption is realistic when the bead moves at a constant speed with respect to the fluid.
+When oscillating a bead however, this frictional force becomes frequency dependent.
+
+Equations have been derived by other authors which provide frequency dependent drag coefficients :cite:`tolic2006calibration`.
+These equations can be used to fit a hydrodynamically correct power spectrum:
+
+.. math::
+
+    P_{hydro}(f) = \frac{D Re(\gamma / \gamma_0)}{\pi^2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+where
+
+.. math::
+
+    f_{c, 0} = \frac{\kappa}{2 \pi \gamma_0}
+
+and
+
+.. math::
+
+    f_{m, 0} = \frac{\gamma_0}{2 \pi m}
+
+with :math:`m` the mass of the bead and :math:`\gamma` the frequency dependent drag coefficient.
+Assuming a measurement deep in bulk, these are given by:
+
+.. math::
+
+    \gamma_{stokes} = \gamma_0 \left(1 + (1 - i)\sqrt{\frac{f}{f_{\nu}}} - \frac{2}{9}\frac{f}{f_{\nu}} i\right)
+
```

Comment:
First you say \gamma, then \gamma_{stokes}.
It is obvious from the text that these are the same, but to be mathematically accurate, I would either use just one of those, or say \gamma = \gamma_{stokes}

Reply:
Yeah you raise a fair point about all the funky `gamma`'s. I tried to keep it in line with the paper as much as possible, to not confuse people who want to compare this stuff to the paper.

The `gamma`'s you mention are only the same deep in bulk. The final `gamma` is the "real" one. All the others are approximations indicated by their subscript (0 being zero frequency / frequency independent drag, stokes being stokes drag only, and the final gamma being the actual one).

`gamma_0` is the drag coefficient for zero frequency and without any surface effect.

`gamma_{stokes}` is the frequency dependent `gamma`.

`gamma` is the full drag coefficient. This includes frequency dependence and a height effect.

If you are deep in bulk then `gamma` = `gamma_{stokes}`. 

I've given it some more thought and changed it to the following to clarify that a bit more:

```
Assuming a measurement deep in bulk, then :math:`\gamma` = :math:`\gamma_{stokes}` which is given by:
```

Then later as the more complex effects are introduced, it is narrowed down to the final `gamma`:
```
This approximation is reasonable, when the bead is far from the surface.
When approaching the surface, the drag experienced by the bead depends on the distance between the center of the bead and the nearest surface of the flow cell.
An approximate expression for the frequency dependent drag coefficient is then given by :cite:`tolic2006calibration`:
```

Do you think that'd be sufficiently clear?

Reply:
Ah, I understand it much better now. I didn't realize the gamma's with the subscripts were all approximations. The current version is sufficiently clear.

Reply:
Ah yes, that sounds much better! Thanks!